app/src/main/res/values/strings.xml

```diff
@@ -31,7 +31,7 @@
     <string name="tree">tree</string>
 
     <string name="sync_blocked" translatable="false">Sync Blocked</string>
-    <string name="sync_started">Sync Started</string>
+    <string name="sync_started">Sync Started, please wait</string>
```

Comment:
Great job on the PR! However, I noticed that you added the phrase "Sync Started, please wait" in English to the strings.xml file, but did not provide translations for this addition in the other language files.

For consistency and to ensure a smooth user experience for non-English speakers, could you please add the corresponding translations for "Sync Started, please wait" in the other language files as well? This will help maintain the app's localization standards.

Thank you for your attention to this!

Best regards,